src/lib.rs

```diff
@@ -378,27 +378,28 @@
 //!
-//! Since Rust 0.7 the standard library had a `time` module. It moved to `libextra`, and then to a
-//! `libtime` library shipped alongside the standard library. In 2014 the work on chrono started in
-//! order to provide a full-featured date and time library in Rust. Some improvements from chrono
-//! made it into the standard library, notably the `chrono::Duration` type was moved to
-//! `std::time::Duration` ([rust#15934]).
+//! Rust first had a `time` module added to `std` in its 0.7 released. It later moved to
+//! `libextra`, and then to a `libtime` library shipped alongside the standard library. In 2014
+//! work on chrono started in order to provide a full-featured date and time library in Rust.
+//! Some improvements from chrono made it into the standard library; notably, `chrono::Duration`
+//! was included as `std::time::Duration` ([rust#15934]) in 2014.
 //!
 //! In preparation of Rust 1.0 at the end of 2014 `libtime` was moved out of the Rust distro and
-//! into the `time` crate to eventually be redesigned ([rust#18832], [rust#18858]), just like the
-//! `num` and `rand` crates. Naturally chrono gained `time` as a dependency. `time` also started
-//! re-exporting `std::time::Duration`. Later the standard library was changed to have a more
-//! limited unsigned `Duration` type ([rust#24920], [RFC 1040]), while the `time` crate kept the
-//! full functionality with `time::Duration`. And `time::Duration` was part of the public API of
-//! chrono.
+//! into the `time` crate to eventually be redesigned ([rust#18832], [rust#18858]), like the
+//! `num` and `rand` crates. Of course kept its dependency on this `time` crate. `time` started
+//! re-exporting `std::time::Duration` during this period. Later, the standard library was changed
+//! to have a more limited unsigned `Duration` type ([rust#24920], [RFC 1040]), while the `time`
+//! crate kept the full functionality with `time::Duration`. `time::Duration` had been a part of
+//! chrono's public API.
 //!
 //! By 2016 `time` 0.1 lived under the `rust-lang-deprecated` organisation and was not actively
-//! maintained ([time#136]). Chrono absorbed the platform functionality and `Duration` type of the
-//! time crate in [chrono#478] (the work started in [chrono#286]). But because of the
-//! `time::Duration` type and to not make this a breaking change `time` would still remain as a
-//! dependency. Crates could opt to use chrono without the `old-time` feature to drop the
-//! dependency. During this time @jhpratt would take over `time` and release what amounts to a new
-//! crate under time 0.2.
+//! maintained ([time#136]). chrono absorbed the platform functionality and `Duration` type of the
+//! `time` crate in [chrono#478] (the work started in [chrono#286]). In order to preserve
+//! compatibility with downstream crates depending on `time` and `chrono` sharing a `Duration`
+//! type, chrono kept depending on time 0.1. chrono offered the option to opt out of the `time`
```

Comment:
chrono -> Chrono

Reply:
IMO "chrono" is a name and as such it shouldn't get capitalized even when it appears at the start of a sentence.

Reply:
Okay :+1:. I also wouldn't mind if we started capitalizing Chrono everywhere.